yggdrasilffi/src/json_api.rs

```diff
@@ -0,0 +1,618 @@
+use std::{
```

Comment:
There are no real changes here. This is breaking out the contents of "lib" into its own space. No constants/functions were harmed in the making of this file